### ISSUE TITLE
Toast text has random codes in lots of places

### ISSUE THREAD

**Issue by adi2412** [adi2412/shopmates#9]
The toast text shows the latitude and longitude in the Track Shopper screen. 

When you tap enter your order, it shows some random number.

When there is some parse backend error, it directly displays that(while logging in or placing order)

This needs to be fixed. Do not show such stuff, and instead show a user friendly message

**Comment by tusharsircar95** [comment]
Yes! Some places I have displayed stuff for debugging purposes. Most of them can be removed like the latitute longitude thing. That is not an error.
However on calling parse API there is a callback function. In case there is an error there I have displayed the error returned by parse. That can be changed to a simple message like an error occurred or something.

**Comment by adi2412** [comment]
I see. We can get this removed later one then. As for the parse callback errors, we need to have an idea of the common errors that can come and provide user friendly messages for them.
